dotnet/Paymentsense.Connect-e.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
this file seems to be specific for visual studio IDE, if it's the case can we remove it, please?

Reply:
this is solution file, you need this file, if you check any .net project in dojo it will have solution file, vast majority of .net developers use visual studio, it is really rare for .net devs to develop .net in visual studio code

Reply:
`you need this file` so if it's specific for the visual studio IDE, why do I need it? IDE preferences shouldn't be checked  in as doing it we are telling the developers what they should use and how, changing their IDE preferences, etc... which makes no sense to me. 

anyways, this is just a sample code and I am not a .net developer, so I am just gonna ignore it as this approach seems to be a common practise in the .net community from what you said.

Reply:
no, this file is not specific for the visual studio IDE, you can open solution in rider and I assume other IDEs but not visual studio code